pyproject.toml

```diff
@@ -84,20 +84,33 @@ build-backend = "setuptools.build_meta"
 # TODO build binaries inside musllinux/manylinux
 # skip = ["*-musllinux_*", "*-win32", "*-manylinux_i686"]
 # Building for these platforms is enough since we are using abi3 packages
-build = ["cp37-macosx_x86_64", "cp37-win_amd64", "cp37-manylinux_x86_64", "cp37-manylinux_aarch64"]
+build = [
+    "cp37-macosx_x86_64",
+    "cp37-win_amd64",
+    "cp37-win32",
+    "cp37-manylinux_x86_64",
+    "cp37-manylinux_aarch64",
+]
 before-all = "make preprocess"
 test-requires = "pytest"
 test-command = "python -bb -m pytest {project}/tests/unittest"
 test-extras = ["test"]
 test-skip = "pp*"
 
+# configure cibuildwheel to build native archs ('auto'), and some emulated ones
+[tool.cibuildwheel.linux]
+archs = ["auto", "aarch64"]
+
 [tool.cibuildwheel.macos]
 before-all = "gmake preprocess"
 
-
 [tool.isort]
 profile = "black"
 
```

Comment:
could we drop `isort` in favor of `ruff`?

Reply:
Yes, I'm using ruff actually.